### PR TITLE
Reseting branch for api key

### PR DESCRIPTION
**Pull request by SeokSong**

### CODE REVIEW

**Comment by SeokSong** [comment]
I accidently put my api in 1 of my merges because my git ignore accidently had a space so I disabled my key I will make a new key lol

File: my-app/src/css/display.css [hunk 1]
```diff
@@ -15,68 +17,62 @@ body, html {
...
-  left: 59px;
-  top: 83px;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 50px;
-  color: #000000;
-  text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
-  margin-top: 25px;
-  margin: 55px;
+  align-items: center;
```
Comment (SeokSong) [inline my-app/src/css/display.css]: This actually goes with Button Span but git is showing its apart of WhatToEat which has been moved

**Comment by SeokSong** [comment]
Sorry about this commit, its honestly a very messy commit history

File: my-app/src/Map.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
+import { useSearchParams } from 'react-router-dom';
+import React, { useState } from 'react';
+import { GoogleMap, useJsApiLoader , Marker} from "@react-google-maps/api";
+import './css/Map.css';
+
+
+
+
+function Map(){
```
Comment (kyuhyunp) [inline my-app/src/Map.js:9]: Nit: Let's use lowercase for the filename and function names.

File: my-app/src/Map.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+  const [latitude, setLatitude] = useState(0);
```
Comment (kyuhyunp) [inline my-app/src/Map.js:11]: Question for my own understanding: what are lines 11 and 12 doing?

File: my-app/src/Map.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+  const [longitude, setLongitude] = useState(0);
+  const [searchParams] = useSearchParams();
+  
+  const nearbySearch = () => {
+    let currentPos = new window.google.maps.LatLng(latitude, longitude);
+    var service = new window.google.maps.places.PlacesService(document.getElementById('map2'));
```
Comment (kyuhyunp) [inline my-app/src/Map.js:17]: Nit: Apparently, var is scoped to the entire function body while let and const are scoped to the next enclosing bracket. So, let's try to reduce the scope of the variable as much as possible and use immutable variables if we can.

File: my-app/src/Map.js [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+    service.nearbySearch(request, function(results, status) {
+        console.log("Call Google nearbySearch API")
+        const restaurantList = document.getElementById('nearbyRestaurants');
+        restaurantList.textContent = "";
+        restaurantList.append(document.createElement("br"));
+        restaurantList.append(document.createElement("br"));
+        if (status === window.google.maps.places.PlacesServiceStatus.OK) {
+          console.log("Google nearbySearch API loaded successfully")
+          for (let i = 0; i<4 && i <results.length; i++){
```
Comment (kyuhyunp) [inline my-app/src/Map.js:33]: Very minor: the spacing in this line is not consistent. For example, i<4 and i <results.length

File: my-app/src/WhatToEat.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import './css/WhatToEat.css';
```
Comment (kyuhyunp) [inline my-app/src/WhatToEat.js:1]: Nice!

**Review by kyuhyunp — APPROVED** [review]
Looks good to me overall!

File: my-app/src/Map.js (hunk 2 above)
Comment (kyuhyunp) [reply my-app/src/Map.js:9]: I don't think I followed camelCase for function name before but I will change this too lol

**Pull request merged**